CONTRIBUTING.md

```diff
@@ -168,14 +214,25 @@ git commit
 
 You have to commit the changes to `go.mod` and `go.sum` before submitting the pull request.
 
+### Running Tests
+
+* Thanos provides make commands that help you run the tests locally.
+* If you have a decent hardware to run the tests, you can run them locally.
+* If you want to run the tests once in a while, it is suitable for you to send in a PR, the built in CI/CD setup runs the tests for you, which is nice for once in a while run.
+*  `make test` - Runs all Thanos Go unit tests against each supported version of Prometheus. This excludes tests in `./test/e2e`.
+*  `make test-local`  Runs test excluding tests for ALL  object storage integrations.
+*   `make est-e2e`    Runs all Thanos e2e docker-based e2e tests from test/e2e. Required access to docker daemon.
+*   `make est-e2e-local`  Runs all thanos e2e tests locally.
+
+
 ### Advanced testing
 
 At some point during development it is useful, in addition to running unit or e2e tests, to run and play with Thanos components manually. While you
 can run any component manually by crafting specific flags for a test setup, there are already some nice tools and scripts available.
 Consider the following methods:
 
-* [quickstart.sh](https://github.com/thanos-io/thanos/blob/b08c0ea62abfe4dcf1400da0e37598f0cd8fa8cf/scripts/quickstart.sh): this script spins
-up a simple example setup. Do `make build` before running the script to build the `thanos` binary first.
+* `make quickstart`: this command spins
+up a simple setup of all Thanos components. Run `make build` before running the script to build the `thanos` binary first.
```

Comment:
`build` is actually a dependency of the `quickstart` target so I think we can remove this sentence.

Reply:
Done